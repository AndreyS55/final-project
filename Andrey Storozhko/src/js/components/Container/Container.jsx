import React from 'react';
import { Route, Switch } from 'react-router-dom';
import Header from '../Header/Header';
import AllPokemonsContainer from '../../containers/AllPokemonsContainer';
import CaughtPokemonsContainer from '../../containers/CaughtPokemonsContainer';
import styles from './Container.scss';

const Container = () => (
  <div className={styles.pokedexContainer}>
    <Header />
    <Switch>
      <Route exact path="/" component={AllPokemonsContainer} />
      <Route path="/caught" component={CaughtPokemonsContainer} />
    </Switch>
  </div>
);

export default Container;
