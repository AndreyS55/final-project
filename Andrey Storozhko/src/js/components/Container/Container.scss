@import '../../../styles/environment/mixins';

.pokedexContainer {
  flex: 1 0 auto;
  display: flex;
  flex-direction: column;
  background: fixed url("../../../img/mainBackground.png") center no-repeat;

  @include tablets() {
    background: #e8e8e8;
  }

  @include phones() {
    background: #e8e8e8;
  }
}