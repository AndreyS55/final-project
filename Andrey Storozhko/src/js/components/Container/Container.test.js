import React from 'react';
import Container from './Container';

it('should render a Container', () => {
  const component = shallow(<Container />);
  expect(component).toMatchSnapshot();
});
