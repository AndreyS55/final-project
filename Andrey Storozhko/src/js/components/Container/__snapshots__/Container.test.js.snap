// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Container 1`] = `
<div
  className="pokedexContainer"
>
  <Header />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/caught"
    />
  </Switch>
</div>
`;
